usr/share/rear/verify/BORG/default/400_check_archive_access.sh

```diff
@@ -0,0 +1,45 @@
+# Query Borg server for repository information
+# and store it to BORGBACKUP_ARCHIVE_CACHE.
+# This should avoid repeatedly querying Borg server, which could be slow.
+
+borg_archive_cache_create && return
+
+LogPrint "Failed to list Borg archive."
+LogPrint "If you decide to continue, ReaR will partition your disks, but most probably will NOT be able to restore your data!"
+Log "Command \"borg list $BORGBACKUP_OPT_REMOTE_PATH ${borg_dst_dev}${BORGBACKUP_REPO}\" returned: "
+borg_list
+
+rear_workflow="rear $WORKFLOW"
+choices[0]="View '$rear_workflow' log file ($RUNTIME_LOGFILE)"
+choices[1]="Use Relax-and-Recover shell and return back to here"
+choices[2]="Continue '$rear_workflow'"
+choices[3]="Abort '$rear_workflow'"
+
+prompt="Do you want to continue?"
```

Comment:
The `prompt` value is shown for each `while ... UserInput` loop
in contrast to the `LogPrint "Failed to list Borg archive."`
that scrolls away at the top.
Therefore the `prompt` value should show the reason of the
current `UserInput` dialog, for example something like
`prompt="Make Borg archive manually accessible"`